src/components/DynamicFormContainer/DynamicForm/AttachmentDropzone/AttachmentDropzone.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FunctionComponent, useCallback, useState } from "react";
+import { useDropzone } from "react-dropzone";
+import { FileUploadType } from "../../../../types";
+import { Button } from "../../../../UI/Button";
+import { FilesInfo } from "./FilesInfo";
+
+const MAX_FILE_SIZE = 20971520;
+
+interface AttachmentDropzone {
+  acceptedFormat: string;
+  onUpload: (processedFiles: FileUploadType[]) => void;
+  onRemove: (fileIndex: number) => void;
+  uploadedFiles: FileUploadType[];
+}
+
+export const AttachmentDropzone: FunctionComponent<AttachmentDropzone> = ({
+  acceptedFormat,
+  onUpload,
+  onRemove,
+  uploadedFiles,
+}) => {
+  const [fileSizeError, setFileSizeError] = useState(false);
+
+  const onDrop = useCallback(
+    async (files: File[]) => {
+      let totalSize = uploadedFiles
+        ? uploadedFiles.reduce((acc: number, current: FileUploadType) => acc + current.size, 0)
+        : 0;
+
+      files.reduce((acc: number, current: File) => (totalSize += current.size), 0);
+
+      if (totalSize > MAX_FILE_SIZE) return setFileSizeError(true);
+
+      const processedFiles = await Promise.all(files.map(processFiles));
+      onUpload(processedFiles);
+      setFileSizeError(false);
+    },
+    [onUpload, uploadedFiles]
+  );
+
+  const removeFile = (fileIndex: number): void => {
+    onRemove(fileIndex);
+    setFileSizeError(false);
+  };
+
+  const { getRootProps, getInputProps, isDragActive, fileRejections } = useDropzone({
+    onDrop,
+    accept: acceptedFormat,
```

Comment:
I think this can be optional if the IT staffs wants to restrict file format. React dropzone seemed to allow this to be empty also.

Reply:
ok

Reply:
yup so i made it that if the schema doesn't have any accept, it will be default to "" and the react dropzone will allow all file types.